terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -673,6 +673,20 @@ def qq(*words):
                         'resources': [
                             '*'
                         ]
+                    },
+
+                    # Extend boundary to allow the service Lambda role to assume
+                    # itself
```

Comment:
Consider dropping the fluff "Extend boundary to" and enhancing the info comment with a one-liner of the reason why the service Lambda role must assume itself. The modifications to this section inform of the boundary extension.